<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Motro</class>
 <widget class="QWidget" name="Motro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="forward_radioButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>正转</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="reverse_radioButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>反转</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
